client/client.go

```diff
@@ -17,7 +17,16 @@ type Client struct {
 	endpoint string
 }
 
-type Toxic map[string]interface{}
+type Attributes map[string]interface{}
+
+type Toxic struct {
+	Name       string     `json:"name"`
+	Type       string     `json:"type"`
+	Stream     string     `json:"stream"`
```

Comment:
It's fine to leave it as a `string`. The idea being you pass in `""` to `AddToxic()` and then get the default stream of `"downstream"` and we don't send a blank string in the json.